zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -282,45 +283,19 @@ public ZooKeeperMain(ZooKeeper zk) {
     void run() throws IOException, InterruptedException {
         if (cl.getCommand() == null) {
             System.out.println("Welcome to ZooKeeper!");
+            System.out.println("JLine support is enabled");
```

Comment:
This line may be noisy. Can we drop it?

Reply:
There are two reasons to retain this output line.
1) as the comment from https://github.com/apache/zookeeper/pull/1458#discussion_r491420998. I am not sure if we need to give a output while using jline3(BSD license).
2) Earlier zookeeper released versions all have this outpt line, so I retain it to make zkCli output lines looks like it was.

Reply:
@eolivelli should I do any changes?